File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/UsuarioDTO.java
package com.flotix.dto;

/**
 * Objeto que almacena la informacion de los usuarios para su devolucion
 * 
 * @author Flor
 *
 */
public class UsuarioDTO {

	private String id;
	private String nombre;
	private String email;
	private String idRol;
	private RolDTO rol;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getIdRol() {
		return idRol;
	}

	public void setIdRol(String idRol) {
		this.idRol = idRol;
	}

	public RolDTO getRol() {
		return rol;
	}

	public void setRol(RolDTO rol) {
		this.rol = rol;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/UsuarioDTO.cs

namespace Flotix2021.ModelDTO
{
    public class UsuarioDTO
    {
        private string _id;

        public string id
        {
            get { return _id; }
            set { _id = value; }
        }

        private string _nombre;

        public string nombre
        {
            get { return _nombre; }
            set { _nombre = value; }
        }

        private string _email;
        
        public string email
        {
            get { return _email; }
            set { _email = value; }
        }

        private string _idRol;

        public string idRol
        {
            get { return _idRol; }
            set { _idRol = value; }
        }
        
        private RolDTO _rol;

        public RolDTO rol
        {
            get { return _rol; }
            set { _rol = value; }
        }
        
        private string _pwd;

        public string pwd
        {
            get { return _pwd; }
            set { _pwd = value; }
        }
    }
}
